apps/src/templates/instructions/TopInstructions.story.jsx

```diff
@@ -21,9 +22,18 @@ import TopInstructions from './TopInstructions';
  * @param {boolean} options.tts
  */
 const createCommonStore = function(options = {}) {
-  const store = createStore(combineReducers({...commonReducers, isRtl}));
+  const store = createStore(
+    combineReducers({...commonReducers, isRtl, sectionData})
+  );
   const pageConstants = {};
   const instructionsConstants = {};
+  const section = {
```

Comment:
NIT: for consistency, imo would be better to dispatch the `setTtsAutoplayEnabled` action so this story code shows how it's actually used in practice. currently this code could cause future devs to expect the rest of this section data to be available in this component as well.